### DISCUSSION TITLE
Error message in *-result.json contains unicode

### DISCUSSION THREAD

**Discussion by yaroslavchaadaev** [allure-framework/allure2#2176] [Questions & Support] [answered]
Hello!

I'm using jest, jest-circus and allure-jest. 

When my test fails, the error message in *-result.json in the test report contains Unicode text.

For example:

```
"statusDetails":
            {
                "message": "\u001b[2mexpect(\u001b[22m\u001b[31mreceived\u001b[39m\u001b[2m).\u001b[22mtoBe\u001b[2m(\u001b[22m\u001b[32mexpected\u001b[39m\u001b[2m) // Object.is equality\u001b[22m\n\nExpected: \u001b[32m400\u001b[39m\nReceived: \u001b[31m200\u001b[39m"
            }
```

Could you tell me, how to fix this?

And in Allure TestOps it looks like:
![image](https://github.com/allure-framework/allure2/assets/89546981/4d8312c0-1d69-41ac-8aff-370b9ddcd865)

**Comment by epszaw** [comment, marked as answer]
Hello! 

There is a [fix](https://github.com/allure-framework/allure-js/pull/798) which should resolve the problem.

Jest reports error messages with ANSI codes, so I just cut them off to leave only the original error message and trace.

**Reply by yaroslavchaadaev** [reply]
@epszaw Hello! It looks a little better after the release, thanks!

But it's still works strange in `statusDetails.message` into the steps field.

Here is an example on "allure-jest": "^2.10.0"
[Example.json](https://github.com/allure-framework/allure2/files/13727765/Example.json)

**Reply by epszaw** [reply]
Will check as soon as will have time for it :)
If you can, please open an issue in the repository, it'll reduce the possibility, that I forget about the problem
